css/base.css

```diff
@@ -41,20 +41,22 @@ body {
 }
 
 @media all and (min-width: 750px) {
-#nav {
-    display: block;
-    list-style-type: none;
-    width: 30vw;
-    float: left;
-    font-size: 1.3em;
-  }
-
   .navigation {
     float: left;
     margin: 5px;
   }
 
-  .menu {
+  #menu_icon {
+    float:right;
     display: none;
   }
+
+  #nav {
```

Comment:
There should only be no classes or ids in `base.css`. Styling for the smaller pieces of the page belong in `modules.css`.